packages/near-fast-auth-signer-e2e-tests/contracts/FT.sol

```diff
@@ -0,0 +1,92 @@
+// Origal repo: https://github.com/Consensys/Tokens/blob/master/contracts/eip20/EIP20.sol
```

Comment:
so this file is extended version of file from above link? could you add a bit more detail about what has modified/extended and why? (eg. I notice that `mint` doesn't exist on above, but added below)

Reply:
I didn't added any explanation because we will never change this file again. It's just a contract implementing the ERC20 interface for testing. 

The code here was copy pasted from the URL linked, and the mint function was added so we can distribute the tokens as we want for testing. 

I was checking other options to implement this contracts, looks like Open Zeppelin haves a npm package with the implementations for all we need ERC20, ERC721, ERC1155. 

But if it's ok I would like to tackle this in another PR, there is too much work here already. 

That's the package I'm looking to use: https://www.npmjs.com/package/@openzeppelin/contracts